yarn.lock

```diff
@@ -1396,6 +1396,70 @@
     uuid "^8.0.0"
     yup "^0.29.3"
 
+"@backstage/core-api@^0.2.23":
```

Comment:
Hmm this shouldn't be resolved in the lockfile like this.  Something has gone a little awry with the dependencies somewhere. Maybe try remove this block manually and run `yarn install` again. Not sure why this happened.

Reply:
I wonder if this is what is causing the weirdness with the api reports @Rugvip?